Backend/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='chase-car-dashboard-backend',
+    version='5.0.0',
+    packages=['core', 'framework', 'components'],
+    url='badgerloop.org',
+    license='',
+    author='badgerloop',
```

Comment:
A bit nit-picky, but could you move it into [the "Create setup.py" commit](b7095edf292950cc4aec7d5c8fcdaf2c80fdb2a3) so that it never shows up as badgerloop?